client/src/components/Speakers/Speakers.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+
+import './Speakers.css'
+
+import ButtonOne from '../Button/ButtonOne';
+
+import InstagramIcon from '@material-ui/icons/Instagram';
+import FacebookIcon from '@material-ui/icons/Facebook';
+import TwitterIcon from '@material-ui/icons/Twitter';
+import YouTubeIcon from '@material-ui/icons/YouTube';
```

Comment:
Why are we even using Material UI if we are leveraging React Bootstrap?

Reply:
I will check it out